test/sourcemap.coffee

```diff
@@ -63,3 +64,55 @@ test "#3075: v3 source map fields", ->
   v3SourceMap = JSON.parse v3SourceMap
   arrayEq v3SourceMap.sources, ['tempus_fugit.coffee']
   eq v3SourceMap.sourceRoot, './www_root/coffee/'
+
+# Source maps aren't accurate on Node v12 ??
```

Comment:
Care to explain?

Reply:
Node v12 reports the line number as `4` where every other version after correctly reports it as `3`.

Reply:
On further investigation this is only for the error message, the stack is correct. Future versions of node also corrected the line in the error messages as well.